src/main/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptions.scala

```diff
@@ -32,7 +32,6 @@ case class SparkBigQueryOptions(
     credentialsFile: Option[String] = None,
     filter: Option[String] = None,
     schema: Option[StructType] = None,
-    skewLimit: Double = SparkBigQueryOptions.SKEW_LIMIT_DEFAULT,
```

Comment:
Update README.md to document this and add a line to CHANGES.md.

Since we are removing a property this could be viewed as a breaking change, but it shouldn't break users if they continue to specify it so I don't really care about changing the version.

Reply:
Done.